api/src/main/java/org/openmrs/module/legacyui/GeneralUtils.java

```diff
@@ -68,4 +69,49 @@ public static Concept getConcept(String id) {
 		
 		return cpt;
 	}
+	
+	/**
+	 * Checks if current version of openmrs is greater or equal to 2.7.0
+	 * 
+	 * @return true if current version is greater or equal to 2.7.0 and false otherwise
+	 * @since 1.17.0
+	 */
+	public static boolean isTwoPointSevenAndAbove() {
```

Comment:
This is cimplicated and brittle. Why don't you use the same trick of trying to load a class like concept reference range which was added in 2.7?

Reply:
This boolean is used in UI/JSP hence the need to check the version instead of loading a particular class. I think this will be reliable as long as we have the version constant(s).

Reply:
We can still have a boolean with loading a particular class.